config/label.go

```diff
@@ -26,10 +30,27 @@ func NewLabelConfig(APIAddress string) *LabelConfig {
 	}
 }
 
+// NewKubeLabelConfig gets a new LabelConfig for Kubernetes
+func NewKubeLabelConfig(APIAddress string, KubeConfig string, KubeNamespace string) *LabelConfig {
+	return &LabelConfig{
+		APIAddress:    APIAddress,
+		KubeConfig:    KubeConfig,
+		KubeNamespace: KubeNamespace,
+	}
+}
+
 // GetApps retrieves task config from the server using the API, and then gets scaling parameters from labels using MicroBadger
 func (l *LabelConfig) GetApps(userID string) (tasks []*demand.Task, maxContainers int, err error) {
 	tasks, maxContainers, err = api.GetApps(l.APIAddress, userID)
 	for _, task := range tasks {
```

Comment:
Does this mean the user no longer configures the image name in the MiaB UI? So the image name is not included in the api.GetApps (i.e. the response from MiaB API)?